CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Fidel Android SDK Change log
 
+## 2.0.0
+- Removed card scanning
```

Comment:
I think we should point to the website `Releases` pages in these changelogs, so that we don’t change details of every release in 2 places.

Reply:
Good point, I will make the change

Reply:
shall we do the same for the README?

Reply:
Done!